gemrb/core/Strings/StringConversion.cpp

```diff
@@ -26,77 +26,65 @@
 
 namespace GemRB {
 
+static bool IsBigEndian() {
```

Comment:
we already have this function somewhere... not able to check where ATM.

Reply:
It's inside of the `DataStream` but in there. Can look for a better position for both if you like.